.devcontainer/devcontainer.json

```
@@ -0,0 +1,72 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/python-3
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": {
+			// Update 'VARIANT' to pick a Python version: 3, 3.6, 3.7, 3.8, 3.9
+			"VARIANT": "3.8",
```

```suggestion
			"PYTHON_VERSION": "3.8",
```
If the variable is changed, it needs to be updated here as well

---

yes, so pls do it on all places :]